include/libdnf/rpm/package.hpp

```diff
@@ -120,14 +120,6 @@ class Package {
     // @replaces libdnf:libdnf/hy-package.h:function:dnf_package_get_group(DnfPackage * pkg)
     std::string get_group() const;
 
-    /// @return If package is installed, return get_install_size(). Return get_package_size() otherwise.
-    /// @since 5.0
-    //
-    // TODO(dmach): drop and use get_install_size()/get_package_size() instead?
-    // @replaces dnf:dnf/package.py:attribute:Package.size
-    // @replaces libdnf:libdnf/hy-package.h:function:dnf_package_get_size(DnfPackage * pkg)
-    unsigned long long get_size() const;
```

Comment:
I am not sure whether we can consider it as a good move. As you can see, the logic from public API was moved to Daemon part and probably we will copy it also into microdnf part.

Reply:
As an alternative we can think about a rename. What do you think?

Reply:
This is part of another PR (#1198) as I stated in the description.
I strongly prefer to remove this method due to its ambiguity.

Reply:
You are correct in principle, but we have a use case that requires it. It means that we need it.